File: LV.2/숫자의 표현/solution.py
def solution(n):
    sum = 0
    count = 0
    for k in range(1,n+1):
        sum = sum + k
        if (n-sum-k)%k == 0:
            count = count + 1
            if sum > n:
                break
    count = count -1
    return count

File: LV.2/숫자의 표현/Solution.java
class Solution {
    public int solution(int n) {
        int sum = 0;
        int count = 0;
        
        for (int k=1; k < n+1; k++) {
            sum += k;
            if (sum > n) {
                break;
            } 
            if ((n-sum-k)%k == 0) {
                count = count +1;
            } 
        }
        return count;
    }
}
